src/log/log.go

```diff
@@ -51,23 +51,24 @@ type Logger struct {
 	mu     sync.Mutex // ensures atomic writes; protects the following fields
 	prefix string     // prefix to write at beginning of each line
 	flag   int        // properties
-	out    io.Writer  // destination for output
+	//out    io.Writer  // destination for output
+	outs   []io.Writer// destinations for ouput
 	buf    []byte     // for accumulating text to write
 }
 
-// New creates a new Logger. The out variable sets the
-// destination to which log data will be written.
+// New creates a new Logger. The outs variable sets the
+// destinations to which log data will be written.
 // The prefix appears at the beginning of each generated log line.
 // The flag argument defines the logging properties.
 func New(out io.Writer, prefix string, flag int) *Logger {
-	return &Logger{out: out, prefix: prefix, flag: flag}
+	return &Logger{outs: []io.Writer{out}, prefix: prefix, flag: flag}
 }
 
-// SetOutput sets the output destination for the logger.
+// SetOutput sets the output destinations for the logger.
```

Comment:
Create a new function named `AddOutput`?

Reply:
No, I just forgot to restore the notes.